types/react-dev-utils/InlineChunkHtmlPlugin.d.ts

```diff
@@ -5,7 +5,7 @@ import HtmlWebpackPlugin = require('html-webpack-plugin');
  * This Webpack plugin inlines script chunks into `index.html`.
  */
 declare class InlineChunkHtmlPlugin extends webpack.Plugin {
-    constructor(htmlWebpackPlugin: HtmlWebpackPlugin, tests: ReadonlyArray<RegExp>);
+    constructor(htmlWebpackPlugin: typeof HtmlWebpackPlugin, tests: ReadonlyArray<RegExp>);
```

Comment:
It's accessing the prototype method (`getHooks`) instead of the instance method.